README.md

```diff
@@ -1,17 +1,41 @@
-# .NET Core
+# .NET Core Libraries (CoreFX)
 
 [![Build status][build-status-image]][build-status]
 
 [build-status-image]: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/badge/icon
 [build-status]: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/
 
-This repository contains the class libraries for .NET Core. This is a
-work in progress, and does not currently contain the entire set of libraries
-that we plan on open sourcing. Make sure to watch this repository in order to be
-notified as we make changes to and expand it. Check out this [blog post] that
-explains our OSS strategy and road map in more detail.
+The corefx repo contains the library implementation (called "CoreFX") for [.NET Core](http://github.com/dotnet/core). It includes System.Collections, System.IO, System.Xml and many other components. It builds and runs on Windows. You can ['watch'](https://github.com/dotnet/corefx/subscription) the repo to see Linux and Mac support being added over the next few months.
 
-Today, the repository contains the following components:
+.NET Core is a modular implementation of .NET that can be used as the base stack for a wide variety of scenarios, today scaling from console utilities to web apps in the cloud.  You can learn more about .NET Core and how and where you can use it in the [.NET Core is open source][.NET Core oss] blog post. 
```

Comment:
We should also link to the [Introducing .NET Core](http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx) blog post.